Snowcast/ResortsTableViewController.swift

```diff
@@ -30,17 +48,25 @@ class ResortsTableViewController: UITableViewController {
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath)
+        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath) as! ResortTableViewCell
 
-        cell.textLabel?.text = self.resorts[indexPath.row].name
+        cell.resortNameLabel.text = self.resorts[indexPath.row].name
 
         return cell
     }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 112 characters